backend/src/routes/api/validate-isv/validateISV.ts

```diff
@@ -119,15 +118,6 @@ export const runValidation = async (
     });
   });
 
-  // Wait for previous config map to be deleted
-  if (cmName) {
-    await waitOnDeletion(() => {
-      return coreV1Api.readNamespacedConfigMap(cmName, namespace).then(() => {
-        return;
-      });
-    });
-  }
-
```

Comment:
We are no longer removing the CM before validation so no need to wait for that to complete